gui/mozregui/wizard.py

```diff
@@ -228,6 +244,17 @@ def options(self):
         fetch_config.set_repo(options['repository'])
         fetch_config.set_build_type(options['build_type'])
 
+        # create a profile if required
+        launcher_class = LAUNCHER_REGISTRY.get(fetch_config.app_name)
+        launcher_class.check_is_runnable()
```

Comment:
check_is_runnable should not be called here - it does some check to know if a launcher is runnable (for example for fennec - firefoxon android -, ensure that a phone is connected. Just remove that line, this is checked elsewhere.